build/safetyOutlierExplorer.js

```diff
@@ -109,36 +100,28 @@ var safetyOutlierExplorer = (function (webcharts, d3$1) {
 		}
 
 		//delete non-numeric endpoints
-		var numMeasures = allMeasures.filter(function (f) {
-			var measureVals = _this.raw_data.filter(function (d) {
-				return d[config.measure_col] === f;
-			});
+		var numMeasures = allMeasures.filter(f => {
+			var measureVals = this.raw_data.filter(d => d[config.measure_col] === f);
 
 			return webcharts.dataOps.getValType(measureVals, config.value_col) === "continuous";
 		});
 
-		this.raw_data = this.raw_data.filter(function (f) {
-			return numMeasures.indexOf(f[config.measure_col]) > -1;
-		});
+		this.raw_data = this.raw_data.filter(f => numMeasures.indexOf(f[config.measure_col]) > -1);
 	};
 
 	function onLayout() {
-		var _this2 = this;
+		//custom filter behavior           
+		var xColSelect = this.controls.wrap.selectAll(".control-group").filter(f => f.option === "x.column").select("select");
 
-		//custom filter behavior          
-		var xColSelect = this.controls.wrap.selectAll(".control-group").filter(function (f) {
-			return f.option === "x.column";
-		}).select("select");
-
-		xColSelect.on("change", function (d) {
+		xColSelect.on("change", d => {
 			var value = xColSelect.property('value');
 
-			_this2.config.x.column = value;
-			_this2.config.marks[1].per[2] = value;
+			this.config.x.column = value;
+			this.config.marks[1].per[2] = value;
 
 			//DY is a hardcoded variable...
-			_this2.config.x.type = value == "DY" ? "linear" : "ordinal";
-			_this2.draw();
+			this.config.x.type = value == "DY" ? "linear" : "ordinal";
```

Comment:
I'd suggest avoiding this hard code and working time_col[].type in to the settings so that the user can specify what type of axis to use. See [here](https://github.com/RhoInc/safety-outlier-explorer/pull/32#pullrequestreview-16034578) for details.